src/lib/Components/Bidding/Components/Timer.tsx

```diff
@@ -4,7 +4,13 @@ import moment from "moment-timezone"
 import * as PropTypes from "prop-types"
 import React from "react"
 
-enum AuctionTimerState {
+// Possible states for an auction:
+// - PREVIEW: Auction is open for registration but artworks cannot be bid on. This occurs when the current time is before any auction's startAt.
+// - CLOSING: Timed auction has started
+// - LIVE_INTEGRATION_UPCOMING: Auction is open for pre-bidding, live portion has not started
+// - LIVE_INTEGRATION_ONGOING: Live auction is in progress
+// - CLOSED: Auction is over
+export enum AuctionTimerState {
```

Comment:
Nit-pick: When enum values are all the same as their keys, I don’t see any benefit in using an enum over a string union.

Especially not if the value needs to be passed to an API that wants the actual value. I don’t know if that applies in this case, but as it sometimes is the case (eventually) I personally lean towards never doing it.

Reply:
These values don't need to be passed to any kind of API.

I personally found the enum to be _super helpful_ while writing this code (due to intellisense).

Reply:
Oh yes, absolutely that’s what we need, but a string union _should_ provide the same intellisense, did it not?